dotCMS/src/main/java/com/dotcms/publishing/output/TarGzipBundleOutput.java

```diff
@@ -42,10 +42,7 @@ public void create() throws IOException {
     }
 
     public static File getBundleTarGzipFile(final String bundleId) {
-        final String fileName = String.format(
-                ConfigUtils.getBundlePath() + File.separator + bundleId + ".tar.gz"
-        );
-
+        final String fileName = String.format("%s%s%s.tar.gz",ConfigUtils.getBundlePath(),File.separator,bundleId);
```

Comment:
Do we need to sanitize the bundleId here?  Seems like we are creating a file from user input.

Reply:
That's a small code improvement that was brought to my attention by the sonarLint plugin the original code was concatenating string within a call to string.format. 
It is not intended to sanitize anything